Modelica/Resources/C-Sources/zlib/gzguts.h

```diff
@@ -101,6 +101,9 @@
 #  ifdef __MVS__
 #    define NO_vsnprintf
 #  endif
+#  ifndef STDC99
```

Comment:
I notice that we use STDC99 in a few places already, but as far as I can see it isn't standard.

The correct replacement for `#ifndef STDC99` seems to be:
`#if !(__STDC_VERSION__ >= 199901L)`
(slightly odd logic, since undefined macros are treated as 0).
However, that should be a separate PR.

Reply:
`STDC99` is set and used within zlib soures only. Same holds for `NO_vsnprintf`.